File: Said_Coley_Romero/Programas/Ruby/Trabajo_17_de_Junio.rb
puts "Ingrese cantidad de estudiantes:"
e = gets.chomp.to_i

for i in 1..e
  puts "Introduce tu nombre:"
  n4M3 = gets.chomp
  many = n4M3.length - 1
  inVert = ""
  for j in (many..0).step(-1)
    z = n4M3[j]
    inVert = inVert + z
  end
  puts("Tu nombre es #{n4M3} y la forma inversa es #{inVert}")
end

File: Said_Coley_Romero/Programas/Dart/Trabajo_17_de_Junio.dart
import 'dart:io';

void main(){
  print("Ingrese cantidad de estudiantes:");
  int a = int.parse(stdin.readLineSync()!);

  for (int i = 1; i <= a; i++) {
    print("Digite su nombre acontinuacion:");
    String n4M3 = stdin.readLineSync()!;

    int many = n4M3.length;
    String inVert = "";

   for (int j = many - 1; j >= 0; j--) {
      String z = n4M3[j];
      inVert = inVert + z;
    }

   print("Tu nombre es $n4M3 y la forma inversa es $inVert");
  }


}